Docker/startozwdaemon.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+shopt -s nocaseglob
+if [ ! -c "$USB_PATH" ]
+then
+	echo "USB Path \"$USB_PATH\" does not exist or is not a Character Device"
+	exit 128
+else
+	CMD="-s $USB_PATH"
+fi
+
+if [ ! -z "$MQTT_SERVER" ]
```

Comment:
`MQTT_SERVER` is still defined in the `Dockerfile`, check is redundant? (applies to others as well)

Reply:
I don't know how other Docker Images that use this as a base image would be affected by this, hence errr on the side of Caution